test/test_jit.py

```diff
@@ -640,7 +640,7 @@ def scaleshift(x, scale, shift):
             torch.randn(4, dtype=torch.float, device='cuda'),
         ]
         ge = self.checkTrace(scaleshift, inputs)
-        self.assertExpectedGraph(ge.graph_for(*inputs))
+        # self.assertExpectedGraph(ge.graph_for(*inputs))
```

Comment:
Why is this commented out?

Reply:
because in phabricator the diff is covered with linter errors, so I can't actually see what I am doing :)